include/cuco/static_multimap.cuh

```diff
@@ -834,6 +860,28 @@ class static_multimap {
       Key const& k,
       KeyEqual key_equal = KeyEqual{}) noexcept;
 
+    /**
+     * @brief Indicates whether the pair `p` exists in the map.
+     *
+     * If the pair `p` was inserted into the map, `contains` returns
+     * true. Otherwise, it returns false. Uses the CUDA Cooperative Groups API to
+     * to leverage multiple threads to perform a single `contains` operation. This provides a
+     * significant boost in throughput compared to the non Cooperative Group
+     * `contains` at moderate to high load factors.
+     *
+     * @tparam PairEqual Binary callable type
+     * @param g The Cooperative Group used to perform the contains operation
+     * @param k The pair to search for
+     * @param pair_equal The binary callable used to compare input pair and slot content
+     * for equality
+     * @return A boolean indicating whether the input pair was inserted in the map
+     */
+    template <typename PairEqual = thrust::equal_to<value_type>>
+    __device__ __forceinline__ bool pair_contains(
+      cooperative_groups::thread_block_tile<ProbeSequence::cg_size> const& g,
+      value_type const& p,
+      PairEqual pair_equal = PairEqual{}) noexcept;
```

Comment:
```suggestion
      PairEqual pair_equal = PairEqual{}) const noexcept;
```

Reply:
I believe that the line 861 above also needs to be fixed like this. Otherwise, calling the function on const object will cause compile error.